.github/workflows/bazel.yml

```
@@ -21,6 +21,10 @@ jobs:
   bazel:
     name: Bazel graph MVP
     runs-on: blacksmith-4vcpu-ubuntu-2404
+    env:
+      # Blacksmith sticky disks own ~/.cache (bazelisk/disk/repo). uv must
+      # not try to create ~/.cache/uv there — that path is not writable.
+      UV_CACHE_DIR: ${{ runner.temp }}/uv-cache
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)